docs/topics/templates/document-attributes.adoc

```diff
@@ -58,6 +58,8 @@ endif::[]
 :name-landing-page: Welcome
 :name-guide-contrib: Contribution Guide
 :name-guide-spring-boot: {SpringBoot} Runtime Guide
+//FIXME: define unique guide name
+//:name-guide-spring-boot-1-5: {SpringBoot} 1.5 Runtime Guide
```

Comment:
Uncomment, please? IMO this is only used in the guide itself and in the link from the 2.1 guide.